### DISCUSSION TITLE
Hotkey to swap Profile

### DISCUSSION THREAD

**Discussion by thewarrrr** [Rem0o/FanControl.Releases#503] [Ideas]
Hello, I was wondering if it was possible to add hotkey to swap profiles. I have a few diff profiles for different tasks/times (gaming, sleeping, working etc...) and hotkeys would be helpful!

**Comment by Rem0o** [comment]
It's coming. You'll be able to create shortcuts to launch a specific profile and bind those to your streamdeck.

**Comment by Rem0o** [comment]
https://github.com/Rem0o/FanControl.Releases/wiki/Hotkeys---Automate-config-changes-(V136-and-up)
